4.4.2/index.html

```diff
@@ -0,0 +1,46 @@
+
+<!doctype html>
+<html lang="en">
+  <head>
+      <title>Example 1</title>
+      <meta charset="utf-8" />
+      <meta http-equiv="x-ua-compatible" content="ie=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" href="style.css" />
+      <link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.7.0/css/all.css'
+      integrity='sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ'
+      crossorigin='anonymous'>
+    <header>
+      <h1 class="header-text">Redes Sociales</h1>
+    </header>
+
+    <div class="box-container">
+      <div class="facebook-container">
+        <p class="facebook-text">Facebook es una herramienta social que conecta a la gente
+         con sus amigos y otras personas que trabajan, estudian y viven
+         en su entorno</p>
+        <a href="https://www.facebook.com">
+          <li class='fab fa-facebook'></li>
+        </a>
+      </div>
+      <div class="twitter-container">
+        <p class="twitter-text">Twitter es un servicio de microblogging. La red permite enviar
+        enviar mensajes de texto plano de corta longitud, con un m&aacute;ximo de
+        140 caracteres, llamados tweets, que se muestran en la p&aacute;gina
+        principa del usuario</p>
+      <a href="https://www.twitter.com">
+        <li class='fab fa-twitter-square'></li>
+      </a>
+      </div>
+      <div class="instagram-container">
+        <p class="instagram-text">Instagram es un programa o aplicaci&oacute;n para compartir fotos con la que
+        otros usuarios pueden aplicar efectos fotogr&aacute;ficos como filtros, marcos
+        colores retro y vintage, luego pueden compartir las fotografias en diferentes
+        redes sociales </p>
+      <a href="https://www.instagram.com">
+        <li class='fab fa-instagram'></li>
```

Comment:
For the icons why did you use <li> tag instead of <i> tag?

Reply:
I don't get it.

I let it like that since it was like that on the homepage example.